doc/airline.txt

```diff
@@ -700,13 +700,16 @@ with the middle mouse button to delete that buffer.
   nmap <leader>- <Plug>AirlineSelectPrevTab
   nmap <leader>+ <Plug>AirlineSelectNextTab
 
-  Note: Mappings will be ignored within a NERDTree buffer.
+  Note: Mappings will be ignored within "g:keymap_ignored_filetypes".
```

Comment:
please either call that explicitly g:airline#extensions#tabline#keymap_ignored_filetypes or at least remove the `g:` prefix